package.json

```diff
@@ -1825,6 +1873,16 @@
                     }
                 ],
                 "priority": "option"
+            },
+            {
+                "viewType": "jupyter-data-wrangler",
+                "displayName": "Data Wrangler",
+                "selector": [
+                    {
+                        "filenamePattern": "*.csv"
```

Comment:
Are the other file types needed here? Not sure until I read further in the review.

Reply:
Will just be implementing csv for now